### PR TITLE
Show a basic helpful 404 page if no redirect exists.

### PR DESCRIPTION
**Pull request by lazyatom**
This should let people search to find the gem/project they wanted, and give instruction about how to contribute a redirect if they'll like to.

This should hopefully be the very minimum to satisfy #4 , but I wouldn't expect that to be finished without some styling.

### CODE REVIEW

**Comment by tomstuart** [comment]
Great!

jQuery magic looks good. While you’re in there, can you do anything about “Foo project”?

**Comment by lazyatom** [comment]
Ah, that's what I get for slavishly copying text. Hopefully better now!

File: features/step_definitions/redirect_steps.rb [hunk 1]
```diff
@@ -13,3 +13,20 @@
 Then /^the redirect should send me to (.*)$/ do |url|
   expect(last_response.location).to eq url
 end
+
+When(/^an umapped old RubyForge URL is visited$/) do
+  visit 'http://some-arbitrary-unmapped-gem.rubyforge.org'
+end
+
+Then(/^a not found page should be presented$/) do
+  expect(Capybara.current_session.driver.response.status).to eq Rack::Utils.status_code(:not_found)
```
Comment (tomstuart) [inline features/step_definitions/redirect_steps.rb]: I think `expect(page.status_code)` would work here.

File: features/step_definitions/redirect_steps.rb [hunk 2]
```diff
@@ -13,3 +13,20 @@
 Then /^the redirect should send me to (.*)$/ do |url|
   expect(last_response.location).to eq url
 end
+
+When(/^an umapped old RubyForge URL is visited$/) do
```
Comment (tomstuart) [inline features/step_definitions/redirect_steps.rb]: “umapped” is a typo.

File: features/step_definitions/redirect_steps.rb (hunk 1 above)
Comment (lazyatom) [inline features/step_definitions/redirect_steps.rb]: I thought Capybara explicitly didn't expose response codes :-/ But it does! Fixed in 8535e71.

File: features/step_definitions/redirect_steps.rb (hunk 2 above)
Comment (lazyatom) [inline features/step_definitions/redirect_steps.rb]: Fixed in 4ef5833

**Comment by tomstuart** [comment]
This is great. Thanks again James!

**Pull request closed without merging**